aws-cpp-sdk-core/CMakeLists.txt

```diff
@@ -402,6 +397,13 @@ else()
     endif()
 endif()
 
+configure_file("${CMAKE_SOURCE_DIR}/toolchains/pkg-config.pc.in" "${PROJECT_NAME}.pc" @ONLY)
```

Comment:
Not sure what you mean by moved considering that pkg-config pc generation was not part of the core/CMakeLists.txt before.